drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/AbstractDataManagementStrategyTest.java

```diff
@@ -70,6 +72,23 @@ public void getPropertiesToHideListPropertyAssigned() {
         commonGetPropertiesToHideList(true);
     }
 
+    @Test
+    public void getPropertiesToHideListWithSelectedColumnNotInGridModel() {
+        doReturn(true).when(gridColumnMock).isPropertyAssigned();
+        List<String> retrieved = abstractDataManagementStrategySpy.getPropertiesToHide(scenarioGridColumnMock, scenarioGridModelMock);
+        assertTrue(retrieved.isEmpty());
+        reset(scenarioGridModelMock);
```

Comment:
No need to reset a mock at the end of the test

Reply:
Test removed.